include/oneapi/dpl/pstl/iterator_impl.h

```diff
@@ -403,6 +403,10 @@ class transform_iterator
     _Iter __my_it_;
     _UnaryFunc __my_unary_func_;
 
+    static_assert(oneapi::dpl::__internal::__is_const_callable_with_arg_v<
+                      _UnaryFunc, typename std::iterator_traits<_Iter>::reference>,
+                  "Unary Functor is not const callable with the reference type of base iterator.");
```

Comment:
I think `__is_const_callable_with_arg_v` reinvents `std::is_invocable_v`; i.e. it should be `static_assert(std::is_invocable_v<const _UnaryFunc, typename std::iterator_traits<_Iter>::reference>,"...");`

Reply:
Good point.  I'll make this switch.  Thanks for pointing me to that.